src/System.Security.Cryptography.Algorithms/tests/RijndaelTests.cs

```diff
@@ -0,0 +1,299 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Encryption.Rijndael.Tests
+{
+    using Rijndael = System.Security.Cryptography.Rijndael;
+
+    /// <summary>
+    /// Since the Rijndael classes wrap Aes, we only test minimally here since Aes has the full test suite.
+    /// </summary>
+    public class RijndaelTests
+    {
+        [Fact]
+        public static void VerifyDefaults()
+        {
+            using (var alg = Rijndael.Create())
+            {
+                // We use an internal class for the implementation, not the public RijndaelManaged
+                Assert.IsNotType<RijndaelManaged>(alg);
+
+                VerifyDefaults(alg);
+            }
+
+            using (var alg = new RijndaelManaged())
+            {
+                VerifyDefaults(alg);
+            }
+        }
+
+        private static void VerifyDefaults(Rijndael alg)
+        {
+            // The block size differs from the base
+            Assert.Equal(128, alg.LegalBlockSizes[0].MinSize);
+            Assert.Equal(128, alg.LegalBlockSizes[0].MaxSize);
```

Comment:
I don't know what our strategy is for places that we're knowingly deviating from netfx.  Theoretically this test should have a different value when run as core via netfx vs core via coreclr.  But I also don't know what state the test harnesses for core-on-netfx are in, so maybe this is deferrable.  (Similar for the two PNSE's below)